packages/craco/README.md

```diff
@@ -125,13 +129,97 @@ Or build your app:
 $ npm run build
 ```
 
-## Custom location for craco.config.js
+# Configuring CRACO
+## CRACO Configuration Overview 
+Create React App ([CRA](https://github.com/facebook/create-react-app)) is intended to allow people to get started with 
+writing React apps quickly. It does this by packaging several key components with a solid default configuration. 
+
+After some initial experimentation, many people find the default CRA is not quite the right fit. Yet, selecting and a 
+configuring a toolchain featuring all of the components CRA already offers is overwhelming. 
+
+CRACO allows you to enjoy the recognizable project structure of CRA while changing detailed configuration settings of 
+each component. 
+
+>_Read on for more useful introduction to CRACO, or jump directly to 
+>[the CRACO configuration file template](#the-craco-configuration-file-template)_
+
+## Notes on CRA Configurations and Problem Solving  
+Keep in mind that there are _some_ configuration settings available to CRA without CRACO. 
+
+Getting exactly what you want may involve a combination of making changes your CRACO configuration file and by using 
+some of the more limited _but still important_ settings available in Create React App. 
+
+Before jumping into customizing your _CRACO_ configuration, step back and think about each part of the problem you're 
+trying to solve. Be sure to review these resources on the CRA configuration, as it may save you time:
+
+ - [Important Environment Variables that Configure CRA](https://create-react-app.dev/docs/advanced-configuration)
+ - [Learn about using `postbuild` commands in `package.json`](https://stackoverflow.com/a/51818028/4028977)
+ - [Proxying API or other Requests](https://create-react-app.dev/docs/proxying-api-requests-in-development/), or "how 
+ to integrate CRA's dev server with a second backend": 
+ [problem statement](https://github.com/facebook/create-react-app/issues/147)
+ - [Search CRACO issues, for gotchas, hints and examples](https://github.com/gsoft-inc/craco/issues?q=is%3Aissue+sort%3Aupdated-desc)
+
+## A Note on Ejecting CRA
```

Comment:
I am not sure about this section. Seems more like noise to me than actual useful documentation.

Reply:
Moved to beginner section.  I added this section because it seemed like there were more-commonly desired configuration changes. Some of these are not addressed by CRACO or do not need to be. 

CRA documentation can be confusing, because it seems like it is not configurable, but it actually is. So these points were to entice the reader into exploring "other places to look" before trying to do it with craco.